libs/common/src/types/mongo-db-models/tfm-users.ts

```diff
@@ -1,18 +1,37 @@
 import { WithId } from 'mongodb';
 import { TeamId } from '../tfm/team-id';
 import { UnixTimestamp } from '../date';
+import { AuditDatabaseRecord } from '../audit-database-record';
 
-export type TfmUser = WithId<{
-  username: string;
-  email: string;
-  teams: TeamId[];
-  timezone: string;
-  firstName: string;
-  lastName: string;
+/**
+ * These properties are only on non-SSO users,
+ * and can be removed once SSO is permanently enabled.
+ * TODO: DTFS2-6892: Remove NonSSOUserProperties once SSO is permanently enabled
+ */
+type NonSsoUserProperties = {
   salt: string;
   hash: string;
-  status: string;
-  lastLogin?: UnixTimestamp;
   loginFailureCount?: number;
-  sessionIdentifier?: string;
-}>;
+};
+
+type SsoUserProperties = {
+  // Azure Oid will not exist on users that have not logged in via SSO
+  azureOid?: string;
+};
+
+export type TfmUser = WithId<
+  {
+    username: string;
+    email: string;
+    teams: TeamId[];
+    timezone: string;
+    firstName: string;
+    lastName: string;
+    status: string;
+    lastLogin?: UnixTimestamp;
+    sessionIdentifier?: string;
+    // Audit records may not exist on a user if it has not been modified after Audit Logs is released
```

Comment:
What do we mean by "after audit logs is released" please, is this not live already?

Reply:
It is -- this is a comment from elsewhere. Due to us not running any migration scripts, this is still technically optional (as we cannot rely on a user object in the db having been updated since it's release)

Reply:
I think we should just remove the comment @AlexBramhill I think it just adds confusion, for me at least

I.e we don't comment about other optional things like `lastLogin`

Reply:
Lovely, removed